cmake/CMakeLists.txt

```diff
@@ -159,6 +159,10 @@ else (MSVC)
   set(LIB_PREFIX)
 endif (MSVC)
 
+if (UNICODE)
+  add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
It's still used. Without these lines, these macro won't be passed to the compiler, and defining UNICODE=ON has no effect.